defaults/test_default.yaml

```diff
@@ -114,7 +114,7 @@ authenticator_password: ''
 # gemini defaults
 n_test_oracle_db_nodes: 1
 gemini_seed: 0
-oracle_scylla_version: '4.4.7'
+oracle_scylla_version: '4.6.9'
```

Comment:
why not 5.0?

Reply:
seems that @aleksbykov want to backport this to 2022.1

I would recommend switching to oracle 5.0 on master/2022.2/5.1, while doing a separate PRs for 5.1/2022.1

Reply:
agree

Reply:
As we agreed long time ago, we use oldest supporting scylla version